src/themes/money/theme.json

```diff
@@ -1127,6 +1127,10 @@
     "contentSpan": 10
   },
 
+  "imageModule": {
+    "contentSpan": 10
```

Comment:
do we not need contentCols too

Reply:
Personally I think it's weird for this or `richTextModule` to have `contentCols`. We should 99%, if not 100%  of the time be using the values stored in the grid variant (which it should use by default) as these align to designs.

Reply:
@jonathan-fielding `contentCols` was hardcoded but I've updated it to read from the theme and updated this PR accordingly too.

@barneyfoxuk @jonathan-fielding Can we merge now and change it later? Money is the only theme that is overriding the default so it will be easy to change later.

Reply:
@peterhorne yes fine by me 👍